tmt/steps/report/__init__.py

```diff
@@ -9,14 +9,20 @@
 from tmt.options import option
 from tmt.plugins import PluginRegistry
 from tmt.steps import Action
+from tmt.utils import field
 
 if TYPE_CHECKING:
     import tmt.cli
 
 
 @dataclasses.dataclass
 class ReportStepData(tmt.steps.StepData):
-    pass
+    include_output_log: bool = field(
+        default=True,
+        option=('--include-output-log / --no-include-output-log'),
+        is_flag=True,
+        show_default=True,
+        help='Include full standard output in resulting xml file.')
```

Comment:
So this new option is relevant for `junit` and `polarion` but not other report plugin, right? This means that `display` or `html` plugin help now suggests this is also supported but in fact it is not. I guess this could be confusing. What about adding the option only to plugins which actually support it?

Reply:
This is problematic down the line since I'm using this objects type at one place, but I'll try to improve it. Maybe add common parent data class for junit and polarion since they share this argument.

Reply:
Ouch, I missed that, +1. This belongs to `junit` plugin, maybe also to `polarion`, but not to shared `report` base.

Reply:
it needs to be in both, but since I'm calling junit from polarion then types don't really match so the check there needs to change, but IMO it should be fairly easy, having it in two places is fine as long as typing stops complaining (I'll make it stop complaining)

Reply:
Yay, trial and error FTW, fixed in  https://github.com/teemtee/tmt/pull/3003/commits/4bcab7f01f50c20ee5da353142b388c5a2b26f85

Reply:
Don't worry about copy & paste: there's at least one other shared option, one telling a plugin which file to save output into, `junit` and `html` have this one, so solving this issue is on my TODO list. Providing some classes to add shared options, to avoid duplication, is the current idea, and I can easily resolve not just the output file, but also this one when the time comes.